phpstan.neon.dist

```diff
@@ -5,28 +5,17 @@ parameters:
 	level: 5
 	inferPrivatePropertyTypeFromConstructor: true
 	paths:
+		- %currentWorkingDirectory%/pwa.php
 		- %currentWorkingDirectory%/wp-admin
 		- %currentWorkingDirectory%/wp-includes
 		- %currentWorkingDirectory%/bundled-theme-support
 		- %currentWorkingDirectory%/integrations
 	bootstrapFiles:
 		- %currentWorkingDirectory%/tests/static-analysis-stubs/wordpress-defines.stub
```

Comment:
`- %currentWorkingDirectory%/vendor/php-stubs/wordpress-stubs/wordpress-stubs.php` is necessary of you don't use my package

Reply:
@westonruter Now we are here :)